ledger-core/v2/runner/machinetype/init.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package machinetype
```

Comment:
1. there is no initialization in this file - please rename it to machine.go
2. I'd suggest to rename package to just "machine"